### ISSUE TITLE
Sorry if this is the wrong place to ask but, is anyone still working on modding tools, Osiris or others?

### ISSUE THREAD

**Issue by TheProgrammer163** [Wunkolo/Thoth#12]
I'm constantly making maps and custom gametypes in Halo 5 and having access to some modding tools to change things like gametypes would be incredibly useful to me.

I've tried with Cheat Engine but I could never find out how to change the default starting weapons of a custom gametype with it for example.

I tried finding the old Halo 5 Modding discord server but the invitation link was outdated.

I tried using Exuberant Tool to put objects outside of parralax but I cannot save the map as a new map unfortunately.

...

TL;DR: is anyone still working on modding tools for Halo 5, or simply modding Halo 5 at all and how can I reach them?

Also, as my username suggest, in theory I could make the tools up to date (I'm not that familiar with C++ however).

The Halo 5 Modding subreddit is dead so you're my last hope

**Comment by Wunkolo** [comment]
I think this project and much of the Halo 5 forge scene and much of the Halo Modding scene in general has moved onto Master Chief Collection ever since it came out and kinda unified everyone's efforts under one IP.
Don't think Halo 5 Forge has a scene anymore in particular to speak of unfortunately so this project is pretty dead. I'll likely archive it now that I'm being reminded of it.
There are lots of discords out there that house people in the modding community that were originally places for ElDewrito/ElDorito and such that have repurposed themselves into MCC communities.
Here's a good one: https://discord.gg/trvWkDGZ6k
